### DISCUSSION TITLE
Todo App: API com ASP.NET Core, CQRS e EF Core - Aula  Design By Contracts

### DISCUSSION THREAD

**Discussion by AndreBarboza17** [balta-io/forum#149] [Tira Dúvidas] [answered]
Bom dia.

Estou tentando reproduzir o conteúdo do curso, porém não estou conseguindo seguir na aula de Design By Contracts, não estou conseguindo importar a referÊncia para a classe [Contract] dentro de Flunt.Validations.

Já tentei utilizar a versão do pacote 1.0.5 que foi mencionado em outro tópico aqui mas não funcionoui.

Estou utilizando a versão do ASP NET 7.

Segue como esta a referencia do pacote no csproj do projeto Todo.Domain.

![image](https://user-images.githubusercontent.com/59001728/206450223-362fb4a7-c38b-4c60-85e9-cd2e80250e45.png)

![image](https://user-images.githubusercontent.com/59001728/206450493-c97383e1-0e09-4a0b-963a-bc5f8ba17f3f.png)

**Comment by andrebaltieri** [comment, marked as answer]
@AndreBarboza17 você executou um `dotnet restore` depois de alterar a versão do pacote?
Se sim, executa também um `dotnet clean` e `dotnet build` novamente.

**Reply by AndreBarboza17** [reply]
Puts, encontrei o erro e não estava na referencia do pacote e sim no nome do namespace (Todo.Domain.Commands.Contract) Onde o correto seria (Todo.Domain.Commands.Contracts) por isso não conseguia importar a classe de Contract do pacote Validations.

Obrigado pela resposta

**Reply by andrebaltieri** [reply]
Relaxa, xará! É assim mesmo... as vezes passam coisas bobas durante o dev... é normal! Só continue estudando e evoluindo!!!!
